datasets/roosendaal/time_curves/energy_extraction_uranium_oxide_time_curve.csv

```diff
@@ -1,42 +0,0 @@
-year,preset_demand
```

Comment:
Is there a reason these files are removed?

Reply:
Yes Roosendaal is double

Reply:
So I left the bd_roosendaal and removed the other

Reply:
Cool! 👍 would love it, if these sort of things can be done in a separate pull request in the future, but I'm fine with this for now.